bpmn-model/src/main/java/io/zeebe/model/bpmn/builder/zeebe/MessageBuilder.java

```diff
@@ -31,10 +31,13 @@ public MessageBuilder name(final String name) {
     return this;
   }
 
-  public MessageBuilder zeebeCorrelationKey(final String zeebeCorrelationKey) {
+  public MessageBuilder zeebeCorrelationKey(final String correlationKey) {
     final ZeebeSubscription subscription = getCreateSingleExtensionElement(ZeebeSubscription.class);
-
-    subscription.setCorrelationKey(asZeebeExpression(zeebeCorrelationKey));
+    subscription.setCorrelationKey(correlationKey);
     return this;
   }
+
+  public MessageBuilder zeebeCorrelationKeyExpression(final String correlationKey) {
```

Comment:
Maybe rename to `correlationKeyExpression` to follow naming conventions used elsewhere